atomate/vasp/firetasks/parse_outputs.py

```diff
@@ -97,6 +97,7 @@ def run_task(self, fw_spec):
                           parse_dos=self.get("parse_dos", False),
                           parse_potcar_file=self.get("parse_potcar_file", True),
                           bandstructure_mode=self.get("bandstructure_mode", False),
+                          parse_bader=self.get("parse_bader", BADER_EXE_EXISTS),
```

Comment:
I believe it is turned on by default as long as bader.exe is installed. I've had problems with it either taking a long time or failing, and causing an otherwise good calculation not to be parsed, so I wanted to be able to disable bader parsing in my workflow definition. I'm going to add some lines about it to the docstring which I realize I forgot to do.

Reply:
Ok, it might be good to change this to an env check variable at some point.